src/newsApi.js

```diff
@@ -0,0 +1,23 @@
+const axios = require("axios")
+
+class NewsApi{
+	constructor(){
+		this.apiKey = process.env.API_KEY
+	}
+
+	fetchNews(callback){
+		console.log('fetching the news...');
```

Comment:
Same as above. Use of console log may be unnecessary now the project is finished.